source/frontend/src/features/properties/map/acquisition/common/models.ts

```diff
@@ -95,8 +95,24 @@ export class OwnerAddressFormModel {
   streetAddress3?: string;
   municipality?: string;
   postal?: string;
-  provinceId?: Api_CodeType;
-  countryId?: Api_CodeType;
+  provinceId?: NumberFieldValue;
+  countryId?: NumberFieldValue;
+
+  static addressLines(apiAddress: OwnerAddressFormModel | undefined): number {
+    if (!apiAddress) {
+      return 1;
+    }
+
+    if (apiAddress.streetAddress3?.trim() !== '') {
+      return 3;
+    }
+
+    if (apiAddress.streetAddress2?.trim() !== '') {
```

Comment:
hmm, not sure if this is a realistic concern but null or undefined are not equal to '', so those would trigger an additional address line